conv_rho.py

```diff
@@ -3,82 +3,110 @@
 
 from optparse import OptionParser
 
-import __main__ as main
-import matplotlib.pyplot as plt
 import numpy as np
 
-import ppafm as PPU
-import ppafm.fieldFFT as fFFT
-import ppafm.GridUtils as GU
+from ppafm import GridUtils
```

Comment:
Why remove the alias on the import? What logic does it use to decide when an alias is valid or not? Would it replace `import numpy as np` with `import numpy`?

Reply:
That was the change requested by flake that I did manually.

I think I will revert everything I did manually to give one a better overview of what precisely black did.

Reply:
ok, I removed this change in [95ff0c7](https://github.com/Probe-Particle/ppafm/pull/96/commits/95ff0c76457fdf14d7de479e0204114becd64254) to create less confusion.